### PR TITLE
Added documentation for caching a query

### PR DESCRIPTION
**Pull request by dongido001**

### CODE REVIEW

**Comment by tbarn** [comment]
hey @dongido001! thanks for this!

I'm going to review it tomorrow and see if I have any feedback before merging.

**Comment by dongido001** [comment]
Alright.

File: README.md [hunk 1]
```diff
@@ -218,6 +218,49 @@ $client = KeenIOClient::factory([
 $results = $client->getSavedQuery(['query_name' => 'total-API-requests']);
 ```
 
+#### Using Cached queries
+
+By [Caching a Query](https://keen.io/docs/api/?php#creating-a-cached-query), you are updating the query you have saved earlier by adding a `refresh_rate` property.
```
Comment (tbarn) [inline README.md]: You can create a cached query from the start too. You don't have to update an existing saved query to do it. There might be a way to make this more clear here.

Comment (tbarn) [reply README.md]: Caching a query isn't updating it. Caching a query is setting the refresh_rate through updating it.

**Comment by dongido001** [comment]
@tbarn that's true.

I've updated the guide again. can you make another pass on it?

**Comment by tbarn** [comment]
Thanks, @dongido001 for working on this! 🎉 Please fill out this form for your shirt: https://goo.gl/forms/w3HK9LWeau1UbZwl2

**Comment by dongido001** [comment]
Thank you!.

**Pull request merged**